pkg/client/multi.go

```diff
@@ -62,6 +62,34 @@ func isNil(obj kclient.Object) bool {
 	return obj == nil || reflect.ValueOf(obj).IsNil()
 }
 
+func onOneList[T any, V ObjectPointer[T]](ctx context.Context, factory ProjectClientFactory, name string, cb func(name string, client Client) ([]T, error)) ([]T, error) {
+	var (
+		result      []T
+		projectName = ""
+	)
+	i := strings.LastIndex(name, "/")
```

Comment:
What is name? I see this var name is used occasionally but its not intuitive to me for what this is.